source/agora/test/Base.d

```diff
@@ -1758,7 +1821,7 @@ private immutable(Block)[] generateBlocks (
 
     foreach (_; 0 .. count)
     {
-        auto txs = blocks[$ - 1].spendable().map!(txb => txb.sign());
+        auto txs = blocks[$ - 1].spendable().takeExactly(4).map!(txb => txb.sign());
```

Comment:
I'm not sure I understand this change. Why are only 4 transactions made?

Reply:
The idea is to leave some unspent outputs in the genesis block to use for enrolling the rest of the validators and possible other testing.